prediction_market_agent/agents/microchain_agent/market_functions.py

```diff
@@ -123,19 +124,20 @@ def __init__(
     def description(self) -> str:
         return self._description
 
+    @observe(name="PredictProbabilityForQuestion")
```

Comment:
I created an issue in Microchain repo, to see if they are interested in having Langfuse support: https://github.com/galatolofederico/microchain/issues/16

If yes, I'll implement it, but in the meantime, track at least these predictions.